### ISSUE TITLE
changing disable/read-only options is not working

### ISSUE THREAD

**Issue by margielm** [angular-slider/ngx-slider#220]
when re-enabling the slider (either through disabled, or readonly properties) the UI is getting proper styling, but the it is not reacting to users interactions. It just stays disabled.

You can observe this behaviour on demo page  :
here for disabled: https://angular-slider.github.io/ngx-slider/demos#disabled-slider
and here for read-only: https://angular-slider.github.io/ngx-slider/demos#read-only-slider

edit:

I tried updating from 2.0.1 to 2.0.2 but still the same.

**Comment by margielm** [comment]
I guess that `disabled` and `readOnly`options are missing  from this list:
```
  private getOptionsInfluencingEventBindings(options: Options): boolean[] {
    return [
      options.draggableRange,
      options.draggableRangeOnly,
      options.onlyBindHandles,
      options.keyboardSupport
    ];
  }
```

**Comment by piotrdz** [comment]
I knew that I would break something with new release :)

Fixed now in master, awaiting next release.

**Comment by piotrdz** [comment]
Released in v2.0.3. Closing.
